src/template-parameters.json

```diff
@@ -479,35 +469,15 @@
             "paramName": "method",
             "type": "EQUALS",
             "paramValue": "convertedObjectIDsAfterSearch"
-          },
-          {
-            "paramName": "method",
-            "type": "EQUALS",
-            "paramValue": "viewedObjectIDs"
           }
         ]
       },
       {
-        "type": "SIMPLE_TABLE",
+        "type": "TEXT",
         "name": "objectData",
         "displayName": "Object Data",
-        "help": "A list of data containing additional information about associated objects (maximum of 20).",
+        "help": "A list of <a href=\"https://www.algolia.com/doc/api-reference/api-methods/added-to-cart-object-ids/#method-param-objectdata-2\">data</a> containing additional information about associated objects (maximum of 20).",
         "enablingConditions": [
-          {
```

Comment:
Similar to the above, if we allow `objectIDs` on a method, I think we should also allow `objectData`. It's just an unfortunate design that these two fields are split apart (instead of something like a single field named `objects`), and so I think we should always pair the two together.

Reply:
Per my comment above, I think you've accidentally removed `objectID` from valid methods above.

My thoughts still stand here (objectID and objectData are the same thing so should always be paired together), but we can probably not make my comment here non-blocking. If users end up asking to send `objectData` on these use cases, we can always add it.

So feel free to resolve this comment.

Reply:
Ah, good catch. 🤦‍♂️